
package com.laegler.openbanking.soap.service;

import javax.xml.ws.WebFault;


/**
 * Internal Server Error
 *
 * This class was generated by Apache CXF 3.3.3
 * 2019-10-21T07:01:27.395+13:00
 * Generated source version: 3.3.3
 */

@WebFault(name = "PUT_Updatethebusinessorthetechnicalcontactsforthegivenorganisation_500", targetNamespace = "http://laegler.com/openbanking/soap/model")
public class PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500 extends Exception {

    private java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500;

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500() {
        super();
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500(String message) {
        super(message);
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500(String message, java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500) {
        super(message);
        this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500 = putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500;
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500(String message, java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500, java.lang.Throwable cause) {
        super(message, cause);
        this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500 = putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500;
    }

    public java.lang.Object getFaultInfo() {
        return this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation500;
    }
}
